src/Styleguide/Components/AuctionCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import { Responsive } from "../Utils/Responsive"
+import { Card } from "../Elements/Card"
+import { Flex } from "../Elements/Flex"
+import { Serif, Sans } from "@artsy/palette"
+import styled from "styled-components"
+import {
+  space,
+  width,
+  SpaceProps,
+  WidthProps,
+  height,
+  HeightProps,
+} from "styled-system"
+
+interface ImageWrapperProps extends HeightProps {}
+const ImageWrapper = styled.div.attrs<ImageWrapperProps>({})`
+  ${height};
+`
+
+interface ArtworkImageProps extends SpaceProps, WidthProps, HeightProps {
+  src: string
+}
+const ArtworkImage = styled.img.attrs<ArtworkImageProps>({})`
+  ${space};
+  ${width};
+  ${height};
+`
+
+const ScaledArtworkImage = props => (
```

Comment:
I'm not sure I like this whole thing. In the designs it looks like the consistent value for the image is the width, but if I use width then it messes up the alignment of the image with the text. Scaling images is hard.